.github/workflows/build_and_push_latest_image.yml

```diff
@@ -31,7 +31,6 @@ jobs:
         run: |
           podman --version
           cd Dockerfiles 
-          podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=v1.0.11
+          podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=master
```

Comment:
Do we want to build image from master or from pull request?

Reply:
@lslebodn Since the master would be updated after merge we will need the master to build and push

Reply:
@lslebodn This event runs after the merge happens hence master

Reply:
And why do we need to push image after commit ?
Midstream will use jus tags IIUC.

Reply:
There will be two images, Once will be the latest image which would be generated after every commit. 
the other one would be generated for every tag created. 
In both cases, ci would build an image and push it to the respective repository 
@lslebodn

Reply:
Hence, we have two files, 
.github/workflows/build_and_push_image_on_tag.yml 
.github/workflows/build_and_push_latest_image.yml
@lslebodn

Reply:
I understand what we do in the .github/workflows/build_and_push_image_on_tag.yml, .github/workflows/build_and_push_latest_image.yml

I just do not understand why do we need to build image after each merge/commit to upstream if it will not be used by anything.

Reply:
@samvarankashyap  I still expect answer to my question/comment. (even though pr was closed)

Reply:
@lslebodn  During our conversations with midstream team. They wanted to have an image that has the latest tag which builds and pushes on every commit and also on every tag release we have.

IMO: There might be chances of having bugs that are fixed in the latest commit and delayed due to tag release. 
In those cases, any team can switch to the latest tag to get the latest version of the playbooks. ( This used to happen in my previous project)